File: app/src/main/java/com/example/b4934/ui/activities/MainActivity.java
package com.example.b4934.ui.activities;

import android.app.Activity;
import android.content.Intent;
import android.content.pm.ActivityInfo;
import android.os.Bundle;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.databinding.DataBindingUtil;

import com.example.b4934.R;
import com.example.b4934.databinding.ActivityMainBinding;

public class MainActivity extends Activity {

    private ActivityMainBinding binding;
    private Button play;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
        binding = DataBindingUtil.setContentView(this, R.layout.activity_main);
        initComponents();

    }

    private void initComponents() {
        play = binding.play;
        play.setOnClickListener((cl) -> {
            startActivity(new Intent(MainActivity.this, GameActivity.class));
        });
    }
}

File: app/src/main/java/com/example/b4934/ui/activities/MainActivity.kt
package com.example.b4934.ui.activities

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import androidx.databinding.DataBindingUtil
import com.example.b4934.R
import com.example.b4934.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    lateinit var binding: ActivityMainBinding
    lateinit var play: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.setContentView(
            this,
            R.layout.activity_main
        )
        play = binding.play
        play.setOnClickListener {
            startActivity(
                Intent(
                    this,
                    GameActivity::class.java
                )
            )
        }
    }
}